packages/calypso-products/src/constants/jetpack.ts

```diff
@@ -553,3 +553,46 @@ export const JETPACK_PRODUCT_RECCOMENDATION_MAP: Record< string, string[] > = {
 		JETPACK_TAG_FOR_WOOCOMMERCE_STORES,
 	],
 };
+
+export const JETPACK_PRODUCTS_INCLUDED_IN_PLAN_MAP = {
```

Comment:
I feel like it is better we put this inside the plan-list like below instead of creating another map object. 

https://github.com/Automattic/wp-calypso/blob/add/jp-price-breakdown/ui-logic/packages/calypso-products/src/plans-list.tsx#L2047

Reply:
Yes, I agree with that. Let me move that over. Thank you

Reply:
Done. Thank you.